include/nuttx/arch.h

```diff
@@ -2398,6 +2398,40 @@ int up_rtc_getdatetime_with_subseconds(FAR struct tm *tp, FAR long *nsec);
 int up_rtc_settime(FAR const struct timespec *tp);
 #endif
 
+/****************************************************************************
+ * Name: up_tls_size
+ *
+ * Description:
+ *   Get tls (tdata + tbss) section size
+ *
+ * Returned Value:
+ *   Size of (tdata + tbss)
+ *
+ ****************************************************************************/
+
+#ifdef CONFIG_SCHED_THREAD_LOCAL
+int up_tls_size(void);
```

Comment:
Should we add sizeof(tls_info_s) to the return value?

Reply:
Done